starlite/app.py

```diff
@@ -101,10 +103,6 @@
 """The default OpenAPI config used if not configuration is explicitly passed to the
 :class:`Starlite <.app.Starlite>` instance constructor.
 """
-DEFAULT_CACHE_CONFIG = CacheConfig()
```

Comment:
Why? What reason is there to not init an in memory cache by default? 

There were exactly 0 issues related to this, and no complaints. I don't get the reason for this change.

Reply:
> Why? What reason is there to not init an in memory cache by default? 

Please try to follow the examples and read through the PR carefully. This is not needed anymore and a default in-memory cache is provided in *every* case. I explained elsewhere why it is a very bad idea to use **the same** instance of an in memory cache, which is essentially a dictionary, throughout the whole application.